second_task.py

```diff
@@ -1 +1,7 @@
+#2 task
+eleks_company = ['Maria', 'Max', 'Julia', 'Peter']
+print("Eleks company:", eleks_company)
+toshiba_company = ['Marta', 'Michael', 'Nick']
+toshiba_company.append(eleks_company)
```

Comment:
extend is better here. The append will add all eleks list as the last element of toshiba, but this is a list of names and not a list of names and list.

Also after your manipulations, eleks should not have employees anymore